doc/running.md

```diff
@@ -10,22 +10,52 @@ Install python dependencies with the following command:
 python3 -m venv venv
 source venv/bin/activate
 pip3 install -r requirements.txt
+pip3 install -r requirements.dev.txt
 ```
 
 
 ## Kubernetes setup
 
-You need a kubeconfig file (`~/.kube/config`) with the credentials for your kubernetes cluster.
+To run a Kubernetes cluster locally, the most convenient solution is probably
+`minikube`. Follow the
```

Comment:
'Docker for Mac' is probably more convenient for Mac users, so perhaps mention that as another possible option to explore.